api/src/pcapi/core/offerers/repository.py

```diff
@@ -581,3 +583,17 @@ def get_emails_by_offerer(offerer: models.Offerer) -> set[str]:
     }
     emails.discard(None)
     return emails
+
+
+def find_venues_of_offerer_from_siret(siret: str) -> Tuple[Offerer | None, list[models.Venue]]:
+    siren = siret[:9]
+    offerer = models.Offerer.query.filter(models.Offerer.siren == siren).one_or_none()
+    venues = (
+        models.Venue.query.join(models.Offerer)
+        .filter(models.Offerer.siren == siren, models.Venue.isPermanent.is_(True))
+        .order_by(models.Venue.publicName)
```

Comment:
(bug) `Venue.publicName` n'est pas forcément renseigné. On devrait sûrement utiliser `Venue.common_name` à la place (qui prend `publicName` s'il est rempli, ou `name` sinon).

```suggestion
        .order_by(models.Venue.common_name)
```